zephyr/wrapper.c

```diff
@@ -107,11 +111,20 @@ static void *heap_alloc_aligned(struct k_heap *h, size_t min_align, size_t bytes
 {
 	k_spinlock_key_t key;
 	void *ret;
+#if CONFIG_SYS_HEAP_RUNTIME_STATS
+	struct sys_memory_stats stats;
+#endif
 
 	key = k_spin_lock(&h->lock);
 	ret = sys_heap_aligned_alloc(&h->heap, min_align, bytes);
 	k_spin_unlock(&h->lock, key);
 
+#if CONFIG_SYS_HEAP_RUNTIME_STATS
+	sys_heap_runtime_stats_get(&h->heap, &stats);
+	tr_info(&zephyr_tr, "heap allocated: %x free: %x max allocated: %x",
```

Comment:
I think for "normal" byte-counts decimal sizes are easier to read, i.e. I'd use `%u`. Or at least put `0x` in front or use `%#x`

Reply:
Change to which format both ok, some developer have a brain can quick know real size with hex.
Ok, I can also change it to decimal, it is more straight forward to most people